cookbooks/kinetic_driving_force/simple-subduction.prm

```diff
@@ -0,0 +1,185 @@
+  set Prescribed velocity boundary indicators = top:function, right x:function, left x:function, bottom x:function
+
+  subsection Function
+    set Function expression = 1e-2; -1e-2
+    set Variable names      = x, y
+  end
+end
+
+subsection Boundary composition model
+  set Fixed composition boundary indicators = top, right, bottom, left
+  set List of model names = function
+
+  subsection Function
+    set Function expression = if(y<112e3, 1, 0); 3300
+    set Variable names      = x, y
+  end
+end
+
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = right y:initial lithostatic pressure, left y:initial lithostatic pressure, bottom y:initial lithostatic pressure
+
+  subsection Initial lithostatic pressure
+    set Representative point = 300e3, 0
+  end
+end
+
+subsection Adiabatic conditions model
+  subsection Compute profile
+    set Number of points = 10000
+  end
+end
+
+subsection Gravity model
+  set Model name = function
+
+  subsection Function
+    set Function expression = 0.0; -10.0
+  end
+end
+
+subsection Boundary temperature model
+  set List of model names                   = initial temperature
+  set Fixed temperature boundary indicators = top, left
+end
+
+subsection Initial temperature model
+  set List of model names = adiabatic, function
+
+  subsection Function
+    set Function expression = -500 * exp(-((y-200e3)*(y-200e3)+(x-10e3)*(x-10e3))/(2*35e3*35e3))
+    set Variable names      = x, y
+  end
+
+  subsection Adiabatic
+    subsection Function
+      set Function expression = 0; 3500
+    end
+  end
+end
+
+subsection Heating model
+  set List of model names =  adiabatic heating, shear heating
+end
+
+subsection Initial composition model
+  set List of model names = function
+
+  subsection Function
+    set Function expression = if(y<112e3, 1, 0); 3300
+    set Variable names      = x, y
+  end
+end
+
+subsection Compositional fields
+  set Number of fields            = 2
+  set Names of fields             = X_field, density_field
+  set Compositional field methods = field, prescribed field
+end
+
+subsection Material model
+  set Model name = kinetic driving force
+
+  subsection Kinetic driving force
+    set Kinetic prefactor Q         = 1e-9
+    set Use temperature cutoff      = false
+    set Reaction cutoff temperature = 0.0
+
+    set Transition depths    = 112e3
+    set Viscosity prefactors = 1, 1
+    set Minimum viscosity    = 1e19
+    set Maximum viscosity    = 1e24
+
+    set Data directory = $ASPECT_SOURCE_DIR/cookbooks/kinetic_driving_force/
+    set Data file name = ol-wad-driving-force-profile.txt
+  end
+end
+
+subsection Mesh refinement
+  set Initial adaptive refinement        = 0
+  set Initial global refinement          = 6
+  set Time steps between mesh refinement = 0
+end
+
+subsection Postprocess
+  set List of postprocessors = visualization, velocity statistics, temperature statistics, pressure statistics, depth average, Stokes residual
```

Comment:
For the purposes of the cookbook I would suggest changing the postprocessors as follows:
```suggestion
  set List of postprocessors = visualization, velocity statistics, temperature statistics, pressure statistics, material statistics, depth average
```